test/unit/bootloader/config/base_test.py

```diff
@@ -124,13 +131,10 @@ def test_get_boot_cmdline_initrd_system_is_dracut_with_overlay(
             'splash root=overlay:UUID=uuid'
 
     @patch('kiwi.xml_parse.type_.get_firmware')
-    @patch('kiwi.logger.log.warning')
-    def test_get_boot_cmdline_firmware_ec2_no_uuid(
-        self, mock_log_warn, mock_firmware
-    ):
+    def test_get_boot_cmdline_firmware_ec2_no_uuid(self, mock_firmware):
         mock_firmware.return_value = 'ec2'
-        self.bootloader.get_boot_cmdline()
-        assert mock_log_warn.called
+        with self._caplog.at_level(logging.WARNING):
```

Comment:
I assume we are not validating the log with an assert on purpose because we already have the 100% code coverage requirement. The assert we used to have was kind of a redundant check. Am I right?

Reply:
yes, I have assertions only for those parts where we checked for a specific log message. All those parts where we just expect some logging I only embedded in the with blocks. I think if the code changes and for some reason no log info exists anymore it should not lead the test to fail unless we explicitly tested for an expected message. Do you think this is ok ?

Reply:
Yes I also think this is OK. No need to track in tests all the log calls and its messages.